backend/de.metas.document.archive/de.metas.document.archive.base/src/main/java/de/metas/document/archive/api/IDocOutboundDAO.java

```diff
@@ -22,13 +22,16 @@
  * #L%
  */
 
+import com.google.common.collect.ImmutableList;
 import de.metas.document.archive.DocOutboundLogId;
+import de.metas.document.archive.api.impl.DocOutboundDAO;
```

Comment:
IMHO importing de.metas.document.archive.api.impl.DocOutboundDAO shall not happen.
extract LogWithLines in it's own file or have it defined in interface.